src/Hazelcast.Net.Tests/Networking/NetworkHostnameTests.cs

```diff
@@ -0,0 +1,134 @@
+﻿// Copyright (c) 2008-2022, Hazelcast, Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Hazelcast.Testing;
+using NUnit.Framework;
+using Hazelcast.Core;
+using Hazelcast.Testing.Remote;
+using System.Threading;
+
+namespace Hazelcast.Tests.Networking
+{
+    internal class NetworkHostnameTests : RemoteTestBase
```

Comment:
You should use the already existing `ClusterRemoteTestBase` and override its `RcClusterConfiguration` property. It already provides the `RcClient`, `RcCluster` etc properties. Unless there's a reason not to do so, but I cannot figure it out.

Reply:
Good catch, thanx. I think the PR is not valid anymore.